spec/Section 6 -- Execution.md

```diff
@@ -176,12 +198,13 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
 
 If the operation is a subscription, the result is an _event stream_ called the
 _response stream_ where each event in the event stream is the result of
-executing the operation for each new event on an underlying _source stream_.
+executing the operation’s _root selection set_ for each new event on an
+underlying _source stream_.
 
-Executing a subscription operation creates a persistent function on the service
+Performing a subscription operation creates a persistent function on the service
 that maps an underlying _source stream_ to a returned _response stream_.
 
-Subscribe(subscription, schema, variableValues, initialValue):
+Subscription(subscription, schema, variableValues, initialValue):
```

Comment:
This one is admittedly the weirdest. Regardless of how we net-out on naming, we should rethink the terms used for the phases of subscription